pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -607,6 +607,11 @@ type serviceLoader struct {
 	items     []kapi.Service
 }
 
+// NewServiceLoader returns a new serviceLoader
+func NewServiceLoader(namespace string, lister kclient.ServicesNamespacer) *serviceLoader {
```

Comment:
Is this java-like? I haven't written much to know...

Tbh I prefer having an initializing function than using plain structs which are ugly. What I am not sure of is if we should have them exported or not. So far it seems we should use them only in this package.